nulib/meadow
# Update the OpenSearch ingest pipeline to use the correct model name

# Summary 
Update the OpenSearch ingest pipeline to use the correct model name

# Specific Changes in this PR
- Remove everything up to and including `huggingface/` from indexed model name

# Version bump required by the PR

See [Semantic Versioning 2.0.0](https://semver.org/) for help discerning which is required.


# Steps to Test
Please let end users know what they need to do to test on staging or production

Check the `embedding_model` field of a newly indexed work and make sure it starts with `sentence-transformers` and doesn't include the `stack-s-` stuff.

There will be no noticeable effect in the dev environment, since it doesn't use the same model naming convention.

# :rocket: Deployment Notes

**Note** - __if you check any of these boxes go to the [(always open) `main` <- `staging` PR](https://github.com/nulib/meadow/pulls) and add detailed notes and instructions to help out others who may end up deploying your changes to production__

- Backward compatible API changes
- Backwards-incompatible API changes
  - Run `Meadow.Search.Index.create_ingest_pipeline("dc-v2-work-pipeline")` in Livebook
  - (Optional) Use the [Update by query API](https://opensearch.org/docs/latest/api-reference/document-apis/update-by-query/) to fix the value in all existing documents

# Tested/Verified
